### PR TITLE
Calendar Company Jobs.

### PR DESCRIPTION
**Pull request by Anamika810**

### CODE REVIEW

**Comment by Anamika810** [comment]
I have added the company named "CALENDAR" which is a time-management company owned by JOHN HALL, and according to him, he doesn't hire people based on resumes.

**Comment by Anamika810** [comment]
Label it as "hacktober-fest accepted", if you can.

**Comment by premkiran7** [comment]
> I have added the company named "CALENDAR" which is a time-management company owned by JOHN HALL, and according to him, he doesn't hire people based on resumes.

@Anamika810 When opening a PR, please write a small description and link the issue to which you are submitting the PR. Refer https://github.com/premkiran7/companies-that-hire-without-resumes/pull/2 if you need an example.

Also, I appreciate you for your efforts but I am afraid I cannot merge this PR because Calendar doesn't seem to be having any jobs and based on my research there is no information on any hiring process whatsoever.

**Comment by Anamika810** [comment]
I have also researched well, @premkiran7 .I am sending you some of the screenshots for your reference, where you can see that the job openings are very recent.You can check others on their linkedin, I am mentioning some of them through the screenshot.
Screenshot (10)

File: Calendar/Calendar.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+- https://www.forbes.com/sites/johnhall/2020/01/10/why-i-hire-without-looking-at-a-resume/#be60bdf7cbff
+- https://ideamensch.com/john-hall/
+
+## Social Media
+
+[<img align="left" alt="Calendar | Youtube" width="22px" src="https://cdn.jsdelivr.net/npm/simple-icons@3.7.0/icons/youtube.svg" />][youtube]
+[<img align="left" alt="Calendar | LinkedIn" width="22px" src="https://cdn.jsdelivr.net/npm/simple-icons@3.7.0/icons/linkedin.svg" />][linkedin]
+
+[youtube]: https://www.youtube.com/calendar?sub_confirmation=1
```
Comment (premkiran7) [inline Calendar/Calendar.md:37]:
```suggestion
[youtube]: https://www.youtube.com/calendar
```

File: Calendar/Calendar.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+2018
+
+## Location
+
+Seattle, WA, USA
+
+## About
+
+Calendar is one of many innovative ideas from the team of John Hall. In fact, the company founders have created many other business applications. These include time tracking, project management, invoicing and billing, and payments solutions. These small business tools integrate with the Calendar app for even greater productivity.John Hall's ideology regarding resume and jobs is - "Would I have hired those people if I’d looked over their resumes in advance? Probably. But I didn’t need to, and frankly, I didn’t want to. I have better things to do with my time, and if you’re an entrepreneur, so do you. "
```
Comment (premkiran7) [inline Calendar/Calendar.md:21]: Calendar is one of many innovative ideas from the team of John Hall. In fact, the company founders have created many other business applications. These include time tracking, project management, invoicing and billing, and payments solutions. These small business tools integrate with the Calendar app for even greater productivity. John Hall's ideology regarding resume and jobs is - "Would I have hired those people if I’d looked over their resumes in advance? Probably. But I didn’t need to, and frankly, I didn’t want to. I have better things to do with my time, and if you’re an entrepreneur, so do you."

Comment (premkiran7) [inline Calendar/Calendar.md:21]:
```suggestion
Calendar is one of many innovative ideas from the team of John Hall. In fact, the company founders have created many other business applications. These include time tracking, project management, invoicing and billing, and payments solutions. These small business tools integrate with the Calendar app for even greater productivity. John Hall's ideology regarding resume and jobs is - "Would I have hired those people if I’d looked over their resumes in advance? Probably. But I didn’t need to, and frankly, I didn’t want to. I have better things to do with my time, and if you’re an entrepreneur, so do you."
```

File: Calendar/Calendar.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+Seattle, WA, USA
+
+## About
+
+Calendar is one of many innovative ideas from the team of John Hall. In fact, the company founders have created many other business applications. These include time tracking, project management, invoicing and billing, and payments solutions. These small business tools integrate with the Calendar app for even greater productivity. John Hall's ideology regarding resume and jobs is - "Would I have hired those people if I’d looked over their resumes in advance? Probably. But I didn’t need to, and frankly, I didn’t want to. I have better things to do with my time, and if you’re an entrepreneur, so do you."
+
+## Hiring Process
+
+Missouri,USA
```
Comment (premkiran7) [inline Calendar/Calendar.md:25]:
```suggestion
NA
```

File: Calendar/Calendar.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+- [John Hall](https://twitter.com/calendar/) - Co-Founder
+
+## Founded
+
+2018
+
+## Location
+
+Seattle, WA, USA
```
Comment (premkiran7) [inline Calendar/Calendar.md:17]:
```suggestion
Draper, Utah, USA
```

File: Calendar/Calendar.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+# Calendar
+
+## Website
+
+https://www.calendar.com/
+
+## Key People
+
+- [John Hall](https://twitter.com/calendar/) - Co-Founder
```
Comment (premkiran7) [inline Calendar/Calendar.md:9]:
```suggestion
- [John Hall](https://www.linkedin.com/in/johnhall4/) - Co-Founder
```

**Comment by premkiran7** [comment]
@Anamika810 I actually had to send out a resume for some job posts (e.g Sales Specialist) and additionally had to make lots of changes in the file. I am accepting the PR this time, but I expected a better PR. Thanks anyways.

**Pull request merged**